src/app/pages/OnDemandTvPage/VideoPlayer/index.jsx

```diff
@@ -49,7 +54,7 @@ const VideoPlayer = ({ assetId, masterBrand, imageUrl, isExpired }) => {
     translations,
   );
 
-  if (isExpired) {
+  if (!episodeIsAvailable) {
```

Comment:
Does this mean that if an episode is in the future, we'll now be rendering the dark gray box (instead of nothing). Do we need to run this by UX / Product?

Reply:
Yes, I'll run it past people but I felt that a box would be better than a broken-looking page, plus users will not be able to access future episodes.

Reply:
Spoke to Gavin - users can’t access future episodes as there is no schedule so good to go